tcms/core/tests/test_sendtestemail.py

```diff
@@ -0,0 +1,15 @@
+from django.core.management import call_command
+from django.test import TestCase, override_settings
+import socket
+
+
+class TestSendTestEmail(TestCase):
+    """Test manage.py sendtestemail command"""
+
+    @override_settings(EMAIL_HOST='bogus.nonexistent')
+    def test_send_false_server(self):
+        """test command with fake SMTP server"""
+        with self.assertRaises(socket.gaierror) as serr:
+            call_command('sendtestemail', 'user@server')
+        the_exception = serr.exception
+        self.assertEqual(the_exception.error_code, -2)
```

Comment:
These 2 lines are totally wrong. You can't access block-level variables outside of the block after the block has finished executing.

Reply:
These were from [https://docs.python.org/3.5/library/unittest.html#unittest.TestCase.assertRaises](url) just to try without any expectation for success.